crates/bevy_winit/src/lib.rs

```diff
@@ -32,10 +33,7 @@ use bevy_input::{
 use bevy_math::{ivec2, DVec2, Vec2};
 #[cfg(not(target_arch = "wasm32"))]
 use bevy_tasks::tick_global_task_pools_on_main_thread;
-use bevy_utils::{
-    tracing::{trace, warn},
-    Duration, Instant,
-};
+use bevy_utils::tracing::{error, trace, warn};
```

Comment:
why split this into two places?

Reply:
No reason, time changed a bit (from instant to web_time so I might have messed "use"s a bit